code/modules/admin/verbs/adminpm.dm

```diff
@@ -164,14 +162,9 @@
 	var/ping_link = check_rights(R_ADMIN, 0, mob) ? "(<a href='?src=[pm_tracker.UID()];ping=[C.key]'>PING</a>)" : ""
 	var/window_link = "(<a href='?src=[pm_tracker.UID()];newtitle=[C.key]'>WINDOW</a>)"
 	to_chat(src, "<span class='pmsend'>[send_pm_type][type] to-<b>[holder ? key_name(C, TRUE, type) : key_name_hidden(C, TRUE, type)]</b>: [emoji_msg]</span> [ping_link] [window_link]")
-
-	/*if(holder && !C.holder)
-		C.last_pm_recieved = world.time
-		C.ckey_last_pm = ckey*/
-
 	//play the recieving admin the adminhelp sound (if they have them enabled)
 	//non-admins always hear the sound, as they cannot toggle it
-	if((!C.holder) || (C.prefs.sound & SOUND_ADMINHELP))
+	if((!admin) || (C.prefs.sound & SOUND_ADMINHELP))
```

Comment:
I'm really logic'd out. C.holder is not src, right? So the verb admin, which is based off of src, would not be applicable right?

Reply:
if not then this should be changed and the commit fix should be reverted

Reply:
yeah i think i need to experiment with having "C.check_rights(stuff)" for this instead of src